src/wwwroot/partials/settings/billing.html

```diff
@@ -5,6 +5,18 @@
     </h2>
     <p>{{ 'billing_subtitle' | translate }}</p>
   </div>
+  <div class="section--wrapper billing--plan-name-container">
+    <hr class="default"/>
+    <div>
+      <label class="uppercase">{{ 'plan_text' | translate }}</label>
+      <h4 class="special uppercase">{{vm.planName}}</h4>
+    </div>
+    <div>
+      <label class="uppercase">{{ 'price_text' | translate }}</label>
+      <h4 class="special">${{vm.planPrice}} {{ 'each_month' | translate }}</h4>
```

Comment:
Currency values are [official currency names](https://en.wikipedia.org/wiki/ISO_4217), for example for USA Dollars, USD is the right name.

JP accepted to show that name in place of _dollars_ or _$_.